markymark/Classes/Rules/Inline/ShortLinkRule.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ShortLinkRule.swift
+//  markymark
+//
+//  Created by Jim van Zummeren on 20/07/2021.
+//  Copyright © 2021 M2mobi. All rights reserved.
+//
+
+import Foundation
+
+open class ShortLinkRule: InlineRule {
+    
+    /// Example: <http://www.google.com>
+    open var expression = NSRegularExpression.expressionWithPattern(
+        #"<(.+?)>"#
+    )
+        
+    public func getAllMatches(_ lines: [String]) -> [NSRange] {
+        guard let line = lines.first else { return [] }
+        return getResults(line).map { $0.range }
+    }
+
+    // MARK: Rule
+
+    public func recognizesLines(_ lines: [String]) -> Bool {
+        guard let line = lines.first else { return false }
+        return getResults(line).count > 0
+    }
+
+    public func createMarkDownItemWithLines(_ lines: [String]) -> MarkDownItem {
```

Comment:
Sometimes these are open and something they aren't for all the rules.
Was there a specific reasoning for that?

Reply:
What exactly do you mean? The `open` accessor is only available for classes and determines wether a class can be subclassed or not in comparisson to `public`. For function's there is nothing more public than public